source/characterProcessing.py

```diff
@@ -130,6 +130,7 @@ def getCharacterDescription(locale,character):
 SYMLVL_MOST = 200
 SYMLVL_ALL = 300
 SYMLVL_CHAR = 1000
+SYMLVL_UNCHANGED = -1

```

Comment:
Can these values be turned into an IntEnum `SYMLVL`? Then set the old variables to maintain backwards compatibility with a note to deprecate in 2022.1. It would be good to change new code to use this enum but I don't think it's necessary to change the rest of the codebase until 2022.1.  

 eg
```python
class SYMLVL(IntEnum):
    NONE = 0
    ...
    
SYMLVL_NONE = SYMLVL.NONE
...
```

Reply:
Even if maintaining the old `SYMLVL_*` attributes, there would be discrepancies with `CONFIGURABLE_SPEECH_SYMBOL_LEVELS`, `SPEECH_SYMBOL_LEVELS` and `SPEECH_SYMBOL_LEVEL_LABELS` which would not anymore contain integers but enum members.
Would you still want me to proceed to this change anyway?
Would you please have an example of such a deprecation notice (to keep them coherent and easy to search for)?

Reply:
I don't think this would break any compatibility? IntEnum members behave the same as an int in most aspects, `isinstance(SYMLVL.NONE, int) == True and SYMLVL.NONE == 0`.  We can move this to a new issue if there is risk involved / more research needed. 

All that would be needed is something like:

```python
class SYMLVL(IntEnum):
    NONE = 0
    ...
    
# The following SYMLVL_ constants are deprecated by #11779 but remain to maintain backwards compatibility. Remove these in 2022.1 and replace instances using them with the SYMLVL IntEnum.
SYMLVL_NONE = SYMLVL.NONE
...
```

Reply:
I've changed this myself, pending a review from another NV Access developer.